package.json

```diff
@@ -55,6 +55,17 @@
 					"type": "number",
 					"default": 120,
 					"description": "Timeout for installing .NET in seconds."
+				},
+				"dotnetAcquisitionExtension.existingDotnetPath": {
+					"type": {
+						"version": "string",
+						"path": "string"
+					},
+					"default": {
+						"version": "all",
+						"path": "C:\\Program Files\\dotnet"
```

Comment:
??

Isn't this going to mean you'll try to look here by default?

Reply:
I'll remove this, it's left over from my testing.